rfcs/convergence/slot-null-rendering.md

```diff
@@ -0,0 +1,232 @@
+# RFC: Slot null rendering refactoring
+
+---
+
+_@bsunderhus_
+
+## Summary
+
+This RFC proposes a solution for a recurring problem involving slots with no `children` property into 2 steps:
+
+1. `getSlots` should stop verifying null rendering by the existence of the property `children`
```

Comment:
Maybe this should be obvious (or will become obvious after reading the rest of the RFC), but on initial reading it's not clear to me what "verifying null rendering" means. So I wonder if there's a better way to word that.